R/DataPull.R

```diff
@@ -0,0 +1,57 @@
+.getPreferenceScoreDist <- function(connection = NULL, 
+                                    preferenceScoreDist = NULL,
+                                    targetId,
+                                    comparatorId,
+                                    outcomeId,
+                                    databaseId,
+                                    analysisId) {
+  if (!is.null(connection)) {
+  sql <-   "SELECT *
+            FROM @schema.@preferenceScoreDist
+            WHERE target_id in (@targetId)
+            	and comparator_id in (@comparatorId)
+            	and outcome_id in (@outcomeId)
+            	and database_id in (@databaseId)
+            	and analysis_id in (@analysisId)"
+  
+  schema <- connection@schema
+  sql <- SqlRender::renderSql(sql = sql)
+  sql <- SqlRender::translate(sql, targetDialect = connection@dbms)$sql
```

Comment:
I don't think this works. SqlRender::translate outputs a character string, so calling $sql will throw an error. Was this code tested? Please don't submit untested code.